### PR TITLE
Modified HeapPageId.java

### PR DESCRIPTION
**Pull request by Rithesh17**
There was a small problem with the hashCode() method, so I've modified it.

### CODE REVIEW

**Review by Ram-Aditya — APPROVED** [review]

**Comment by sbshah97** [comment]
Why has this Pull Request not been merged?

**Review by manishsbhoopalam — APPROVED** [review]

**Review by Priiyam — APPROVED** [review]

**Comment by Ram-Aditya** [comment]
@salman-bhai We thought we needed to wait for your approval as we had requested a review from your side.
Sorry about that!We `ll go ahead with the merge now then.

**Pull request merged**
